Editor/Bootstrap/RegionBootstrap.cs

```diff
@@ -11,7 +11,7 @@ namespace AmazonGameLift.Editor
     /// <summary>
     /// Keeps the region UI state and saves it to settings.
     /// </summary>
-    internal class RegionBootstrap
+    public class RegionBootstrap
```

Comment:
Curious why all these internal classes are becoming public? Could you elaborate on whats changed? Same for all other occurrences in this commit.

Reply:
So they changed to public because the only things referencing them were all within the same folder, but as we are using a separate folder structure within resources (Whether this is their finalized position is to be looked). So we need access to the models creation and updating from here.

Reply:
I asked this same question in PR 92, my understanding is Unity compiles the Editor folder into a single assembly, so is `public` actually necessary compared to `internal`?